var/spack/repos/builtin/packages/intel-oneapi-compilers-classic/package.py

```diff
@@ -2,8 +2,8 @@
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
-
-from spack import *
+import os
+from spack.package import *
```

Comment:
```suggestion
import os

from spack.package import *
```

Reply:
I should have read your review before looking at the failures, would've saved me the time of figuring out why isort was mad at me. Thanks!

Reply:
Np